lessons/equivalence-and-laws/langs/en-us/index.adoc

```diff
@@ -0,0 +1,167 @@
+= Equivalence and the Laws of Arithmetic
+
+@description{Students use properties to analyze when equations both with and without variables are true.}
+
+@lesson-prereqs{associative-property}
+
+@keywords{variable}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Determine if two expressions are equivalent by applying Circle Transformations.
```

Comment:
instead of "by applying Circle Transformations" how about "using Circles of Evaluation"